src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatSymb = fromFormat.pop();
+  const toFormatSymb = toFormat.pop();
+
+  const dateSplit = date.split(fromFormatSymb);
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    for (let k = 0; k < toFormat.length; k++) {
+      switch (true) {
```

Comment:
Using a switch statement with 'true' as a condition is not recommended. Use if, else if, and else blocks instead for better readability.